.github/workflows/test.yaml

```diff
@@ -553,6 +554,7 @@ jobs:
       - name: Set up base deps on musl
         if: matrix.metadata.build == 'linux-musl'
         run: |
+            apk update
```

Comment:
Why this? It will usually slow down ci

Reply:
Yep, but at the moment CI is erroring out on musl builds saying you need to run `apk update`. See [my comment](https://wasmerio.slack.com/archives/CDNJPS7SA/p1682942675958989) on Slack.